site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
+
+
+## Before you begin
+
+Please check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and queue configuration.
+
+## 1. MiniCluster definition
+
+Since a Flux MiniCluster uses  a `batch/job` to run the actual workload, no extra components are needed for it to be managed by Kueue, However the following aspects should be taken into account:
```

Comment:
```suggestion
Since a Flux MiniCluster runs as a [`batch/Job`](https://kubernetes.io/docs/concepts/workloads/controllers/job/), no extra components are needed for it to be managed by Kueue. However, the following aspects should be taken into account:
```